marimbabot_command_recognition/msg/Command.msg

```diff
@@ -0,0 +1,8 @@
+
```

Comment:
We may move the messages into our marimbabot_msgs package as well, shouldn't we?

Reply:
Yeah, we can. From my side, it's easy to change and won't cause any problems. Just take care don't have the same name of an ambiguous name of msg files.

Reply:
please just merge this PR, and then I will create a patch to rebase the msg.